File: eKarton/eKarton.Service/Databases/Nalaz.cs
using System;
using System.Collections.Generic;

namespace eKarton.Service.Databases
{
    public partial class Nalaz
    {
        public int NalazId { get; set; }
        public string? Datum { get; set; }
        public string? LicnaAnamneza { get; set; }
        public string? RadnaAnamneza { get; set; }
        public int PacijentId { get; set; }

        public virtual Pacijent Pacijent { get; set; } = null!;
    }
}

File: eKarton/UI/ekarton_mobile/lib/models/nalaz.dart
import 'package:json_annotation/json_annotation.dart';

part 'nalaz.g.dart';

@JsonSerializable()
class Nalaz {
  Nalaz(this.nalazId, this.datum, this.licnaAnamneza, this.radnaAnamneza,
      this.pacijentId);

  int? nalazId;
  String? datum;
  String? licnaAnamneza;
  String? radnaAnamneza;
  int? pacijentId;

  factory Nalaz.fromJson(Map<String, dynamic> json) => _$NalazFromJson(json);

  Map<String, dynamic> toJson() => _$NalazToJson(this);
}
